src/dispatch/static/dispatch/src/router/config.js

```diff
@@ -298,9 +298,15 @@ export const protectedRoute = [
         {
           path: "/:organization/feedback",
           name: "FeedbackTable",
-          meta: { title: "Feedback" },
+          meta: { title: "Incident feedback", group: "feedback" },
           component: () => import("@/feedback/Table.vue"),
```

Comment:
We need to move everything we have in `feedback` to a new subdirectory called `incident`

```suggestion
          component: () => import("@/feedback/incident/Table.vue"),
```